Common/Essential-Studio/licensing/licensing-errors.md

```diff
@@ -11,7 +11,39 @@ documentation: ug
 
 Licensing error popup is displayed with various messages under different circumstances. Here are some ways to resolve different issues.
 
-## Licensing errors before version 20.4.0*
+## Licensing errors from 20.4.0* 
```

Comment:
do we need to mention licensing errors from 20.4.0*? can we mention just **licensing errors**